lib/ansible/config/base.yml

```
@@ -1171,11 +1171,29 @@ DEFAULT_VAULT_IDENTITY:
 VAULT_ENCRYPT_SALT:
   name: Vault salt to use for encryption
   default: ~
-  description: 'The salt to use for the vault encryption. If it is not provided, a random salt will be used.'
+  description:
+    - The salt to use for the vault AES256 encryption method. If it is not provided, a random salt will be used.
+    - This is not recommended to as repeated use of salt/secrets is insecure, but it is here for those that
+      need reproducible outputs, mostly for testing.
+    - This does not affect other methods, only AES256.
   env: [{name: ANSIBLE_VAULT_ENCRYPT_SALT}]
   ini:
   - {key: vault_encrypt_salt, section: defaults}
   version_added: '2.15'
+VAULT_METHOD:
+  name: Vault method to use for encryption
+  default: v2
+  choices:
+    'aes256': AES 256 with PBKDF2HMAC key derivation with double hexlify byte shield
+    'v2': Using cryptography's fernet API with dual keys, AES 128, CBC and sCrypt key derivation, with base64 byte shield
```

We shouldn't need to explain *how* Fernet works. It's sufficient to state we're using it (along with Scrypt):

```suggestion
    'v2': Fernet with dual keys and Scrypt key derivation with a base64 byte shield
```

---

I'm just saving them a lookup by giving the info up front, also matching the other method